CHANGELOG.md

```diff
@@ -4,8 +4,7 @@
 
 ### Added
 
-- [Integration](https://crates.io/crates/mcan-core) with the
-[`mcan`](https://crates.io/crates/mcan) crate.
+- [Integration](https://crates.io/crates/mcan-core) with the [`mcan`](https://crates.io/crates/mcan) crate.
```

Comment:
The `mcan` stuff should not be removed from the changelog

Reply:
It's not. Unnecessary new line has been removed.